.ci.yaml

```diff
@@ -1664,6 +1664,32 @@ targets:
         ["devicelab","hostonly", "linux"]
       task_name: web_benchmarks_canvaskit
 
+  - name: Linux web_benchmarks_ddc
```

Comment:
New test shards must have `bringup: true`. Once they are stable, they will be allowed to become non-bring-up.

Reply:
Done!